pkg/framework/aries/context.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package aries
+
+import "github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
+
+// Context provides handles to various configs to be used by the clients
+type Context struct {
+	outboundTransport transport.OutboundTransport
```

Comment:
In a future PR, it would be nice to get rid of the logging singleton. We could instead pass it around in the Context.
@sudeshrshetty @rolsonquadras